AppCenterDistribute/AppCenterDistribute/Internals/Ingestion/MSDistributeIngestion.m

```diff
@@ -28,14 +28,8 @@ - (NSString *)getHttpMethod {
   return kMSHttpMethodGet;
 };
 
-- (NSDictionary *)getHeadersWithData:(NSObject *__unused)data eTag:(NSString *)eTag {
-
-  // Set Header params.
-  NSMutableDictionary *headers = [self.httpHeaders mutableCopy];
-  if (eTag != nil) {
-    [headers setValue:eTag forKey:kMSETagRequestHeader];
-  }
-  return headers;
+- (NSDictionary *)getHeadersWithData:(NSObject *__unused)data eTag:(NSString *__unused)eTag {
+  return self.httpHeaders;
```

Comment:
Is it related change?

Reply:
No but part of improving coverage as it drops some.

Reply:
And this is dead code.